apollo-router/src/plugins/telemetry/otlp.rs

```diff
@@ -118,19 +126,26 @@ pub(crate) enum EndpointDefault {
 #[derive(Debug, Clone, Deserialize, Serialize, Default, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub(crate) struct HttpExporter {
+    /// Headers to send on report requests
     #[serde(default)]
     pub(crate) headers: HashMap<String, String>,
 }
 
 #[derive(Debug, Clone, Deserialize, Serialize, Default, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub(crate) struct GrpcExporter {
+    /// The optional domain name for tls config.
+    /// Note that domain name is will be defaulted to match the endpoint is not explicitly set.
     #[serde(default)]
     pub(crate) domain_name: Option<String>,
+    /// The optional ca for tls config
     pub(crate) ca: Option<String>,
+    /// The optional cert for tls config
```

Comment:
```suggestion
    /// The optional cert for tls config
```

Is this a path to the file or a string of the contents?

Reply:
Contents, config expansion should be used to inject.